src/main/java/org/sasanlabs/service/vulnerability/fileupload/UnrestrictedFileUpload.java

```diff
@@ -41,11 +43,13 @@
  */
 @VulnerableAppRestController(
         descriptionLabel = "UNRESTRICTED_FILE_UPLOAD_VULNERABILITY",
-        value = "UnrestrictedFileUpload")
+        value = UnrestrictedFileUpload.CONTROLLER_PATH)
 public class UnrestrictedFileUpload {
     private Path root;
     private Path contentDispositionRoot;
+    private List<byte[]> heapMemoryFileStore = new ArrayList<>();
```

Comment:
Instead of this, shall we use the current filesystem like we are doing in all other levels?